plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/BuildUnsignedGramineEnclave.kt

```diff
@@ -44,10 +47,16 @@ open class BuildUnsignedGramineEnclave @Inject constructor(objects: ObjectFactor
     }
 
     private fun generateGramineDirectManifest(templateManifest: String) {
+        /**
+         * It's possible for Gramine to launch threads internally that conclave won't know about!
```

Comment:
`Conclave` upper case is nicer!
Also maybe `doesn't` instead of `won't`?

Reply:
Capitalisation, sure. But I think won't is the right word here. Definitely makes sense for it to be future tense at this point!